docs/content/DimensionSpecs.md

```diff
@@ -154,3 +154,207 @@ Example for the `__time` dimension:
+
+##### csv namespaceParseSpec
+
+|Parameter|Description|Required|Default|
+|---------|-----------|--------|-------|
+|`columns`|The list of columns in the csv file|yes|`null`|
+|`keyColumn`|The name of the column containing the key|no|The first column|
+|`valueColumn`|The name of the column containing the value|no|The second column|
+
+*example input*
+```
+bar,something,foo
+bat,something2,baz
+truck,something3,buck
+```
+
+*example namespaceParseSpec*
+```json
+"namespaceParseSpec": {
+  "type": "csv",
+  "columns": ["value","somethingElse","key"],
+  "keyColumn": "key",
+  "valueColumn": "value"
+}
+```
+
+##### tsv namespaceParseSpec
+
+|Parameter|Description|Required|Default|
+|---------|-----------|--------|-------|
+|`columns`|The list of columns in the csv file|yes|`null`|
+|`keyColumn`|The name of the column containing the key|no|The first column|
+|`valueColumn`|The name of the column containing the value|no|The second column|
+|`delimiter`|The delimiter in the file|no|tab (`\t`)|
+
+
+*example input*
+```
+bar|something,1|foo
+bat|something,2|baz
+truck|something,3|buck
+```
+
+*example namespaceParseSpec*
+```json
+"namespaceParseSpec": {
+  "type": "tsv",
+  "columns": ["value","somethingElse","key"],
+  "keyColumn": "key",
+  "valueColumn": "value",
+  "delimiter": "|"
+}
+```
+
+##### customJson namespaceParseSpec
+
+|Parameter|Description|Required|Default|
+|---------|-----------|--------|-------|
+|`keyFieldName`|The field name of the key|yes|null|
+|`valueFieldName`|The field name of the value|yes|null|
+
+*example input*
+```json
+{"key": "foo", "value": "bar", "somethingElse" : "something"}
+{"key": "baz", "value": "bat", "somethingElse" : "something"}
+{"key": "buck", "somethingElse": "something", "value": "truck"}
+```
+
+*example namespaceParseSpec*
+```json
+"parseSpec": {
+  "type": "customJson",
+  "keyFieldName": "key",
+  "valueFieldName": "value"
+}
+```
+
+
+##### simpleJson namespaceParseSpec
+The `simpleJson` namespaceParseSpec does not take any parameters. It is simply a line delimited json file where the field is the key, and the field's value is the value.
+
+*example input*
+ 
+```json
+{"foo": "bar"}
+{"baz": "bat"}
+{"buck": "truck"}
+```
+
+*example namespaceParseSpec*
+```json
+"namespaceParseSpec":{
+  "type": "simpleJson"
+}
+```
+
+
+#### JDBC namespace update
+The JDBC namespaces will pull from a database cache. If the `tsColumn` is set it must be able to accept comparisons in the format `'2015-01-01 00:00:00'`. For example, the following must be valid sql for the table `SELECT * FROM some_namespace_table WHERE timestamp_column >  '2015-01-01 00:00:00'`. If `tsColumn` is set, the caching service will attempt to only pull values that were written *after* the last sync. If `tsColumn` is not set, the entire table is pulled every time.
+
+```json
+{
```

Comment:
I don't see a "type" in this JSON Object.  Should I have to specify a `"type": "jdbc"` or something for the JDBC namespace?